api/disk.go

```diff
@@ -0,0 +1,27 @@
+package api
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+)
+
+func (api *API) ResizeDisk(instanceID, extraDiskSize int) (map[string]interface{}, error) {
+	data := make(map[string]interface{})
+	failed := make(map[string]interface{})
+	id := strconv.Itoa(instanceID)
+	log.Printf("[DEBUG] go-api::disk::resize instance ID: %s", id)
+	path := fmt.Sprintf("api/instances/%s/disk?extra_disk_size=%d", id, extraDiskSize)
+	response, err := api.sling.New().Put(path).Receive(&data, &failed)
+	if err != nil {
+		return nil, err
+	}
+	if response.StatusCode != 200 {
+		return nil, fmt.Errorf("ResizeDisk failed, status: %v, message: %s", response.StatusCode, failed)
+	}
+
+	if err = api.waitUntilAllNodesReady(id); err != nil {
```

Comment:
Configured seems to never be set to false for the nodes when starting the resize. While the new worker branch sets it to true once completed. So will be consider finished after the 30 second sleep.

Reply:
Sorry, I need to merge some PRs :-)

Reply:
`configured` should now be set to false after submitting disk resize request